qemu/tests/fio_perf.py

```diff
@@ -209,7 +209,7 @@ def fio_install(tarball):
                 if s:
                     raise exceptions.TestFail("Failed to online disk: %s" % o)
     # format disk
-    if format:
+    if format == "True":
```

Comment:
Yanhui, could you please give us more information on "format" error you met? I am trying to retest your situation. The "format" type here is bool no doubt.